pages/views/components/esl-related-target.njk

```diff
@@ -0,0 +1,12 @@
+---
+layout: content
+title: ESL Related Target
+seoTitle: ESL Related Target - mixin to find and control the appropriate element
```

Comment:
It seems to me that `appropriate` is unfortunate word. It would be more appropriate to use `associated` or `linked`.
It's just a title for SEO. We should be bolder and use more synonyms in this field.

Reply:
Thanks!